uploadNeuroDB/HRRT_PET_insertion.pl

```diff
@@ -380,17 +381,15 @@
             $message = "\tERROR: Protocol not found for $minc_file.\n\n";
             # write error message in the log file
             $utility->writeErrorLog(
-                $message, $NeuroDB::ExitCodes::UNKNOW_PROTOCOL, $log_file
+                $message, $NeuroDB::ExitCodes::UNKNOWN_PROTOCOL, $log_file
             );
             # insert error message into notification spool table
             $notifier->spool(
                 'HRRT_PET insertion'   , $message,   0,
                 'HRRT_PET_insertion.pl', $upload_id, 'Y',
                 'N'
             );
-            exit $NeuroDB::ExitCodes::UNKNOW_PROTOCOL;
```

Comment:
If missing execution is aborted despite the fact that the protocol does not seem mandatory for the pipeline to succeed.

Reply:
not sure I understand here. We can talk during our call.